Tests/MessagingPush/MessagingPushImplementationTest.swift

```diff
@@ -42,6 +41,10 @@ class MessagingPushImplementationTest: UnitTest {
         profileStoreMock.identifier = givenIdentifier
         queueMock.addTaskReturnValue = (success: true, queueStatus: QueueStatus.successAddingSingleTask)
 
+        var config = SdkConfig()
+        config.autoTrackDeviceAttributes = true
+        sdkConfigStoreMock.config = config
```

Comment:
This is copy/pasted code in a few of the test functions. Maybe pulling this out into a function in this class would be handy:

```swift
private func enableDeviceAttributes() {
        var config = SdkConfig()
        config.autoTrackDeviceAttributes = true
        sdkConfigStoreMock.config = config
}
```

Reply:
Make sense ! 👍